samples/cpp/visual_language_chat/visual_language_chat.cpp

```diff
@@ -26,16 +49,21 @@ int main(int argc, char* argv[]) try {
 
     pipe.start_chat();
     std::cout << "question:\n";
+
     std::getline(std::cin, prompt);
-    pipe.generate(
-        prompt,
-        ov::genai::image(image),
-        ov::genai::streamer(print_subword)
-    );
+    auto resuls = pipe.generate(prompt, properies);
+    if (generation_config.is_beam_search()) {
```

Comment:
I would keep sample as simple as possible and demonstrated beam search / other types of sampling in dedicated samples